docs/setup_vscode_macos.md

```diff
@@ -0,0 +1,554 @@
+---
+layout: spec
+title: Setup VSCode
+sitemapOrder: 20
+---
+
+Setup up VS Code for C/C++ on macOS
+===================================
+{: .primer-spec-toc-ignore }
+
+[Visual Studio Code](https://code.visualstudio.com/) is a lightweight, easy-to-use, source code editor with debugging support.  It runs on macOS, Windows, and Linux (including CAEN Linux).  Visual Studio Code is not the same program as Visual Studio.
+
+This tutorial is specific to macOS.  Looking for the [Windows version](setup_vscode_wsl.html)?
+
+<div class="primer-spec-callout info" markdown="1">
+If you already have VS Code installed with the C/C++ extensions, skip to the [Create a project](#create-a-project) section.
+</div>
+
+## Prerequisites
+VS Code relies on external command line tools.  To install CLI tools, follow the [macOS command line tools tutorial](setup_macos.html).
+
+Make sure you have a compiler and a debugger installed.  Your version might be different.  Instructions for installation on [macOS](setup_macos.html#install-cli-tools).
+```console
+$ g++ --version
+Apple clang version 13.1.6 (clang-1316.0.21.2.5)
+$ lldb --version
+Apple Swift version 5.6.1 (swiftlang-5.6.0.323.66 clang-1316.0.20.12)
+```
+
+Next, follow our [Command line interface (CLI)](cli.html) tutorial.
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** Make sure you have installed [CLI tools for macOS](setup_macos.html#install-cli-tools) before continuing.
+</div>
+
+## Install
+Make sure you have macOS 11.1 or later.
+```console
+$ sw_vers
+ProductName:	macOS
+ProductVersion:	11.7
+```
+
+Use the homebrew package manager to install VS Code.  You can run this command from any directory.
+```console
+$ brew install --cask visual-studio-code
+```
+
+### Extensions
+Make sure VS Code is installed correctly by checking the version.  You need version 1.52.1 or higher.
+```console
+$ code --version
+1.52.1
+```
+
+Install the Microsoft [C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) and the [CodeLLDB extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).  See the [C/C++ extension alternatives](#cc-extension-alternatives) section for details about why we recommend these extensions.
+```console
+$ code --install-extension ms-vscode.cpptools
+$ code --install-extension vadimcn.vscode-lldb
+```
+
+Restart VS Code.
+
+Verify that the extensions are installed.  It's OK if you have other extensions installed.
+```console
+$ code --list-extensions
+ms-vscode.cpptools
+vadimcn.vscode-lldb
+```
+
+## Create a project
+To create a VS Code project, create a folder (directory).  There are many ways to create folders: Finder AKA File Explorer, VS Code interface, VS Code integrated terminal, and the system terminal.  We'll use the system terminal and call our example project `p1-stats`.
```

Comment:
Perhaps remove "AKA File Explorer", since that's Windows.

Reply:
Fixed 9d20b6ba28920b124871251ec8e955bf40f0b43b